checker/mta_sts.go

```diff
@@ -73,9 +76,12 @@ func getKeyValuePairs(record string, lineDelimiter string,
 	return parsed
 }
 
-func checkMTASTSRecord(domain string) *Result {
+func checkMTASTSRecord(domain string, timeout time.Duration) *Result {
 	result := MakeResult(MTASTSText)
-	records, err := net.LookupTXT(fmt.Sprintf("_mta-sts.%s", domain))
+	ctx, cancel := context.WithTimeout(context.TODO(), timeout)
```

Comment:
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), timeout)
```
Nit, and just curious: is there a particular reason we'd rather use the TODO context?

Reply:
Seems like the do the same thing but `Background` probably has the meaning we want. Thanks!